packages/jaeger-ui/src/constants/index.tsx

```diff
@@ -12,14 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-export const TOP_NAV_HEIGHT = 46 as 46;
+export const TOP_NAV_HEIGHT = 46 as const;
```

Comment:
what's the difference if this simply said `export const TOP_NAV_HEIGHT = 46;`, without `as`?

Reply:
nothing, I just fixed it.